Dockerfile

```diff
@@ -11,6 +11,10 @@ ENV JAVA_HOME /usr/lib/jvm/java-7-openjdk-amd64
 
 ENV MESOS_NATIVE_JAVA_LIBRARY /usr/lib/libmesos.so
 
+ARG MESOS_RELEASE
+ARG STORM_RELEASE
+ARG MIRROR
```

Comment:
@drewrobb Lets set the default values for these variables like @erikdw mentioned?